GuessingGame.html

```diff
@@ -1,26 +1,20 @@
-﻿<!DOCTYPE html>
+<!DOCTYPE html>
 
 <html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+
 <head>
     <meta charset="utf-8" />
     <title>Guess The Word</title>
-    <style>
-        .container {
-            width: 800px;
-            margin: 0 auto;
-        }
-    </style>
+    <link href='http://fonts.googleapis.com/css?family=Ubuntu:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
     <script>
         var secretWords = ["kentucky", "ohio", "indiana", "virginia", "tennesse", "florida", "alabama"];
 
-        function CheckWord()
-        {
+        function CheckWord() {
             var word = document.getElementById('txtGuess').value;
 
-            for (var x = 0; x < secretWords.length; x++)
-            {
-                if(word == secretWords[x])
-                {
+            for (var x = 0; x < secretWords.length; x++) {
+                if (word.toLowerCase() == secretWords[x]) {
```

Comment:
This allows the user to enter any variation of letter casing and the word will still match the value in the array.